content/en/building/training/training-cards/training-cards-configuration.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Training Cards Configuration"
+linkTitle: "Training Cards Configuration"
+weight: 15
+description: >
+  Deploy in-app training cards for remote training.
+keyword: training
+relatedContent: >
+  building/training/training-cards
+  building/training/training-cards-resources/
+  building/examples/training
+  building/examples/learning-care/
+aliases:
+   - /building/training/training-cards/
```

Comment:
I don't think the first `alias` is necessary, as this is the page still exists, but the content was replaced the `Training` content, right? So the readers who have that link should land on the correct page, which is the one you replaced.

Reply:
Removed!